### PR TITLE
skip haYPoint=endpoint when update periodic pings

### PR DESCRIPTION
**Pull request by pablomuri**
HA flow periodic ping is sending a packet out when the Ypoint = Endpoint when periodic ping is activated. This PR skip this kind of haFlow during the periodic ping update and log a warning. 

This functionality must be removed in https://github.com/telstra/open-kilda/issues/5224

### CODE REVIEW

File: src-java/ping-topology/ping-storm-topology/src/main/java/org/openkilda/wfm/topology/ping/bolt/FlowFetcher.java [hunk 1]
```diff
@@ -366,6 +368,12 @@ public void handleOnDemandHaFlowRequest(Tuple input) throws PipelineException {
         emit(input, pingContext, pullContext(input));
     }
 
+    private boolean hasHasOneHaSubFlowWithYPointEqualsEndpoint(HaFlow haFlow) {
+        SwitchId yPointSwitchId = haFlow.getForwardPath().getYPointSwitchId();
+        return haFlow.getHaSubFlows().stream()
+                .filter(subFlow -> subFlow.getEndpointSwitchId().equals(yPointSwitchId)).count() == 1;
```
Comment (dmitrii-beliakov) [inline src-java/ping-topology/ping-storm-topology/src/main/java/org/openkilda/wfm/topology/ping/bolt/FlowFetcher.java:374]: as an option, you can use `Stream::anyMatch` idiom

**Review by dmitrii-beliakov — APPROVED** [review]

**Review by niksv — APPROVED** [review]

**Comment by yuliiamir** [comment]
The issue is still reproduced, additional changes are required.

**Comment by pablomuri** [comment]
@yuliiamir I forgot to add the filter also in the automatic update of the periodic pings from data base. Now it seems to be working fine.

**Comment by yuliiamir** [comment]
Appropriate changes have been added and have been tested manually. 
Checked:
1. There is no traffic(periodical pings) for Ha-Flow on the YPoint when one of the ends is YPoint.
2. Traffic(periodical pings)  has been detected for the Ha-Flow when none of the ends are YPoint.

CorrectBehaviourForHaFlowWithPeriodicPings

**Review by niksv — APPROVED** [review]

**Pull request merged**